### DISCUSSION TITLE
[Bug]: Swiping on mobile incorrectly reduces review item count

### DISCUSSION THREAD

**Discussion by ofcedwards** [blakeblackshear/frigate#12103] [Report a Bug] [answered]
### Describe the problem you are having

When interacting with review items on mobile, swiping left causes the item count displayed to reduce by one but does not remove the item or otherwise appear to mark it as reviewed. A refresh shows the item has in fact been marked reviewed. Because of this, the same item can be swiped on multiple times until the count displayed at the top reduces to zero and finally to a loading icon.

I initially discovered this while trying to find a way to make previews stop playing on mobile. My thought was that swipe right starts it and swipe left ends it, but there does not appear to be a way to make them stop. I can make a separate discussion for that issue if desired.

### Steps to reproduce

1. Navigate to review page on mobile with unreviewed items
2. Swipe left on any review item
3. Continue swiping left on the same review item

### Version

0.14.0-c871beb

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A - Nothing unusual in logs. Nginx logs show an item was marked reviewed on each swipe.
```

### Operating system

UNRAID

### Install method

Docker CLI

### Network connection

Mixed

### Camera make and model

N/A

**Comment by NickM-27** [comment, marked as answer]
thanks, this is a double bug really. after https://github.com/blakeblackshear/frigate/pull/12104 it will stop playback and only mark as reviewed if it is not already reviewed

**Reply by ofcedwards** [reply]
As usual, thanks for the quick fix! Pulled the latest dev image to test and it's now working as expected.
